src/domain/bitcoin/lightning/fee-calculator.ts

```diff
@@ -1,14 +1,14 @@
-import { FEECAP, FEEMIN } from "@domain/bitcoin"
+import { FEECAP_PERCENT, FEEMIN } from "@domain/bitcoin"
 import { toSats } from ".."
 
 export const LnFeeCalculator = (
-  { feeCap = FEECAP, feeMin = FEEMIN } = {
-    feeCap: FEECAP,
+  { FEECAP_PERCENT = FEECAP_PERCENT, feeMin = FEEMIN } = {
```

Comment:
Linter is throwing an error for this. Maybe we can remove `FEECAP_PERCENT` entirely from the constructor here.